transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportConnector.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.trino.spi.connector.Connector;
+import io.trino.spi.connector.ConnectorMetadata;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.connector.ConnectorTransactionHandle;
+import io.trino.spi.function.FunctionId;
+import io.trino.spi.function.FunctionProvider;
+import io.trino.spi.transaction.IsolationLevel;
+import java.io.File;
+import java.io.FileFilter;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import org.apache.bval.util.StringUtils;
+
+import static java.util.Objects.*;
+
+
+/**
+ * This class implements the interface of Connector from Trino SPI as a part of Trino plugin
+ * to load UDF classes in Trino server following the development guideline
+ * in https://trino.io/docs/current/develop/spi-overview.html
+ */
+public class TransportConnector implements Connector {
+
+  private static final Logger log = Logger.get(TransportConnector.class);
+  private static final String DEFAULT_TRANSPORT_UDF_REPO = "transport-udf-repo";
```

Comment:
should we put the default in `TransportConfig` itself?

Reply:
Done in https://github.com/linkedin/transport/pull/133